.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+### PR Summary:
+
+### PR Checklist
+------------
+ - [ ] Includes appropriate unit test(s)
+ - [ ] Appropriate docstring(s) are added/updated
+ - [ ] Code is (approximately) PEP8 compliant
+ - [ ] Issue(s) raised/referenced?
```

Comment:
```suggestion
 - [ ] Issue(s) raised/referenced/closed by this PR?
```

Reply:
Changing the language slightly to "addressed" in order to reflect that a PR is generally opened before closing the PR but more committal than "referenced"